src/util.rs

```diff
@@ -22,9 +22,18 @@ pub fn path_to_bytes<P: AsRef<Path>>(path: &P) -> Option<&[u8]> {
 }
 
 pub fn get_db() -> Result<DB> {
-    let mut path = config::zo_data()?;
-    path.push("db.zo");
-    DB::open(path)
+    // FIXME: fallback to old database location; remove in next breaking version
+    if let Some(home_dir) = dirs::home_dir() {
+        let db_file = home_dir.join(".zo");
+
+        if db_file.exists() && db_file.is_file() {
```

Comment:
The `exists` check can be removed here, since `is_file` does that check anyway.

Reply:
Fixed locally. Once we settle on the schema changes, I'll push along with any other changes that'll need to be made.